docs/man_pages/project/configuration/resource.md

```diff
@@ -0,0 +1,41 @@
+resource
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder resource [<Command>]`
+
+Lists information about splashscreens and icons for Android, iOS and WP8
+
+<% if(isConsole) { %>
+<% if(isMobileWebsite)  { %>
+WARNING: This command and its extended commands are not applicable to mobile website projects. To view the complete help for this command, run `$ appbuilder help generate`
+<% } %>
+<% } %>
+<% if((isConsole && (isCordova || isNativeScript)) || isHtml) { %>
+
+### Attributes
+`<Command>` extends the `resource` command. You can set the following values for this attribute.
+* `create` - For creating icons or splashscreens from a single high-resolution image.
```

Comment:
Creates icons or splash screens for all mobile platforms from a single high-resolution image.